h2o-core/src/main/java/hex/Model.java

```diff
@@ -715,7 +718,10 @@ public void setNames(String[] names, String[] columntypes) {
     public Key _cross_validation_predictions[];
     public Key<Frame> _cross_validation_holdout_predictions_frame_id;
     public Key<Frame> _cross_validation_fold_assignment_frame_id;
-    
+
+    // Key to a TE model
+    public Key<Model> _te_model_key;
```

Comment:
In the middle of nowhere. Let's move it up a little bit or group it with other TE methods and attributes.